### ISSUE TITLE
Bays with name "Bok" in Croatia rendered at low zooms

### ISSUE THREAD

**Issue by Janjko** [ZeLonewolf/openstreetmap-americana#1024]
If you go to this link:

https://zelonewolf.github.io/openstreetmap-americana/#map=6.92/44.005/15.203

and look at Croatia's coast, you will see a lot of "Bok" labels which are a frequent name for a bay in Croatia. Those are all small bays mapped as a node, but they can be visible from zoom 5 upwards which is a lot for a bay which can fit one rowing boat and one swimmer.

This is an interesting example [of a "Bok" bay](https://www.openstreetmap.org/node/1383978703) because just next to it is a [bay called "Jaz"](https://www.openstreetmap.org/node/6984966134) which also has only 3 letters, but somehow, "Bok" bays prevail. Island names or town names should probably be rendered before these little insignificant entities.

But a good thing is that "Bok" is also used as a greeting in Croatian, so we get greeted by your map when we look at it 😄 👋 Bok!

**Comment by 1ec5** [comment]
Very interesting! Thanks to your bug report, I just noticed that a lot of very local bays are being shown at low zoom levels. I opened #1025 separately because I’m unsure whether it’s related or just a coincidence. Wikidata doesn’t seem to know of any famous bay named Bok or Jaz, but it does know of some other kinds of places by these names, such as [a Martian crater](https://www.wikidata.org/wiki/Q4938698). Maybe whatever’s causing this bug can’t tell the difference between planets either!
